projects/ngneat/dialog/src/lib/dialog.component.ts

```diff
@@ -105,23 +105,32 @@ export class DialogComponent implements OnInit, OnDestroy {
 
     backdropClick$.pipe(takeUntil(this.destroy$)).subscribe(this.dialogRef.backdropClick$);
 
-    if (this.config.enableClose) {
-      merge(
-        fromEvent<KeyboardEvent>(this.document.body, 'keyup').pipe(filter(({ key }) => key === 'Escape')),
-        backdropClick$
+    // backwards compatibility with non-split option
+    const closeConfig =
+      typeof this.config.enableClose === 'object'
```

Comment:
null returns true, let's go with the boolean check

Reply:
I mean, thanks to typescript it _shouldn't_ really matter, but you're right, I'll make the check more explicit